drivers/adc-dac/ad74413r/iio_ad74413r_trig.c

```diff
@@ -47,7 +47,7 @@
 /************************ Variable Declarations *******************************/
 /******************************************************************************/
 struct iio_trigger ad74413r_iio_trig_desc = {
-	.is_synchronous = false,
+	.is_synchronous = true,
```

Comment:
This seems like project specific behavior and makes me wonder why iio_ad74413r_trig.c file is where it is.
It seems the ad74413r_iio_trig_desc definition should be in application code and not under drivers/.